src/test/mod.rs

```diff
@@ -344,9 +344,9 @@ fn stdin_formatting_smoke_test() {
     }
 
     #[cfg(not(windows))]
-    assert_eq!(buf, "fn main() {}\n".as_bytes());
+    assert_eq!(buf, "stdin:\n\nfn main() {}\n".as_bytes());
     #[cfg(windows)]
-    assert_eq!(buf, "fn main() {}\r\n".as_bytes());
+    assert_eq!(buf, "stdin:\n\r\nfn main() {}\r\n".as_bytes());
```

Comment:
`writeln!` only emits the line feed character [1], so the first `\r` is unnecessary here.

[1] https://doc.rust-lang.org/std/macro.writeln.html

Reply:
That explains why the windows build was failing 😄. Thanks.

Reply:
Fixed in a759b83